lib/sign_in_page.dart

```diff
@@ -9,29 +11,136 @@ class SignInPage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      body: Column(
+      body: Stack(
         children: [
-          TextField(
-            controller: emailController,
-            decoration: InputDecoration(
-              labelText: "Email",
-            ),
+          Column(
+            children: [
+              Padding(
+                padding: EdgeInsets.only(top: 50.0),
+                child: Container(
+                  height: 150,
+                  width: 150,
+                  decoration: BoxDecoration(
+                    borderRadius: BorderRadius.all(Radius.circular(30.0)),
+                    color: Colors.black12,
+                    image: DecorationImage(
+                      image: AssetImage('assets/images/Logo.png'),
+                    ),
+                  ),
+                ),
+              ),
+              Container(
+                padding: EdgeInsets.only(top: 50.0, left: 20.0),
+                alignment: Alignment.topLeft,
+                child: Text(
+                  "LOGIN",
+                  style: TextStyle(
+                    fontSize: 36.0,
+                    fontFamily: 'Cornerstone',
+                    color: Color(0xFF113654),
+                  ),
+                ),
+              ),
+              Padding(
+                padding: EdgeInsets.only(top: 5.0, left: 20.0, right: 20.0),
+                child: TextField(
+                  controller: emailController,
```

Comment:
On Android and iOS, the phone's keyboard can change slightly to make typing more convenient for the user when it knows what the text field is for. In this text field, we can let the system know that this is an email field by putting `keyboardType: TextInputType.emailAddress,` as one of the properties under `TextField` (on my phone it'll show the "@" key without having to go into the special characters menu when it knows the field is email-related)

Note that you might need to do a soft app restart rather than just a hot reload to see the keyboard difference.